apis/network/v1beta1/internetgateway_types.go

```diff
@@ -43,24 +51,25 @@ func (v *VPCIDReferencerForInternetGateway) Assign(res resource.CanReference, va
 		return errors.New(errResourceIsNotInternetGateway)
 	}
 
-	ig.Spec.VPCID = value
+	ig.Spec.ForProvider.VPCID = value
 	return nil
 }
 
 // InternetGatewayParameters define the desired state of an AWS VPC Internet
 // Gateway.
 type InternetGatewayParameters struct {
```

Comment:
InternetGateway supports tags. You can use `CreateTagsRequest` to add tags.

Reply:
Added this.